source/leuven/services/openondemand.rst

```diff
@@ -0,0 +1,250 @@
+obvious save button on the editor page.
+
+Jobs
+----
+
+All jobs submitted from Open Ondemand will run on wICE. The Genius cluster is not supported. This also means that all your jobs should be submitted as **slurm** jobs.
+For more detail on how to run jobs on wICE, check out our :ref:`quickstart guide <wice_t2_leuven>`_.
+
+The jobs tab has two menus: 'Active Jobs' and 'Job Composer':
+
+Active jobs
+===========
+
+This lists all of your running, queued and completed jobs. Completed jobs will slowly dissapear after a couple of minutes. You have a handy overview of some job
+details by clicking the large arrow next to the job id, including things like the node list, the account you used, the status of the job... It also gives you the
+option to open the folder of the job script in the file manager. and thus inspect the created output and error files. 
+
+If your job is still running, you can also delete it by clicking the bin under 'Actions'. The 'Active jobs' menu does not allow re-submission of your job. How to
+(re-)submit jobs will be made clear in the next chapter.
+
+Job Composer
+============
+
+This part deserves some extra attention, as there is more to it than just simply creating and editing a .slurm file. Under the job composer tab you can find two other 
+menus, namely 'Jobs' and 'Templates'. As templates are the backbone of job creation in Open Ondemand, we will start by explaining this. The 'Jobs' menu is pretty much 
+self-explanitory once understanding this.
+
+Templates
+~~~~~~~~~
+
+To enter the Templates menu, you can click on 'Templates' at the top once you are in the 'Job Composer' menu. You can also access this menu by clicking the button 'New 
+Job'-'From Template'. Once in this menu, you should see a table with 3 System Templates. The resources that are requested in these scripts are the default settings. 
+The templates:
+
+- CPU job template: this is a template for CPU jobs (batch partition). This is also the default template (which you will get when clicking 'From Default Template' under the 'New Job' button in the 'Jobs' menu).
+- GPU job template: a template for creating GPU jobs 
+- Big memory CPU jobs: a template to create job scripts that submit to the big memory partition instead of the default batch partition
+
+You can create your own templates from scratch or by copying one of the existing templates. In both cases you will be redirected to a page where you can provide a
+name, the cluster (only wICE) and add some notes. Once you save this, you will create a new folder in the templates folder 
+``$VSC_DATA/ondemand/data/sys/myjobs/templates/``. This information will be saved in the ``manifest.yml`` file that is always present in each of the template folders. 
+Which other files that are present in this folder will depend on how you created your new template. If you use the 'New Template' button, it will also contain the job 
```

Comment:
```suggestion
Which other files will be present in this folder depends on how you created your new template. If you use the 'New Template' button, it will also contain the job 
```
(makes the sentence a bit easier to read)